apps/e2ee-server/Dockerfile

```
@@ -0,0 +1,26 @@
+
+RUN yarn install --immutable
+
+COPY . .
+
+RUN yarn run build
+
+FROM node:20-alpine
+
+WORKDIR /app
+
+RUN apk add --no-cache tzdata && rm -rf /var/cache/apk/*
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)